packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -72,20 +70,20 @@ const getCidFromV0Transaction = (txResponse: TransactionResponse): CID => {
 }
 
 const getCidFromV1Transaction = (txResponse: TransactionResponse): CID => {
-  const decodedArgs = iface.decodeFunctionData('anchorDagCbor', txResponse.data)
```

Comment:
so I don't *really* know what this is doing, but it looks to me like using a function designed to decode the data here would be less brittle than just directly counting bytes.  Any reason for changing this?

Reply:
Can change it back. I think this is safe though since it's unlikely that EVM will change the number of bytes used for the function signature. If it did it would also break the tool we are using here.